components/autofill/sql/create_shared_schema.sql

```diff
@@ -18,37 +18,23 @@ CREATE TABLE IF NOT EXISTS addresses_data (
     email               TEXT NOT NULL,
 
     time_created        INTEGER NOT NULL,
-    time_last_used      INTEGER,
+    time_last_used      INTEGER NOT NULL DEFAULT 0,
```

Comment:
Asking for my own understanding, what is the value of setting a default value here and not for the other timestamps.

Reply:
Good question: none, there's no good reason for that and it should be made consistent. In practice, we always supply the values which is why it works, so I guess it makes sense to remove the defaults? wdyt?

Reply:
Yep, sounds good!